src/main/java/hbp/mip/repositories/ExperimentRepository.java

```diff
@@ -6,16 +6,16 @@
 import hbp.mip.models.DTOs.UserDTO;
 import hbp.mip.utils.Exceptions.BadRequestException;
 import hbp.mip.utils.Exceptions.ExperimentNotFoundException;
-import hbp.mip.utils.Exceptions.InternalServerError;
 import hbp.mip.utils.JsonConverters;
 import hbp.mip.utils.Logger;
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.rest.core.annotation.RestResource;
 
 import java.util.Date;
 import java.util.UUID;
 
-
+@RestResource(exported = false)
```

Comment:
Isn't there a better way instead of having to add that in every non exposes resource?

Can we do the opposite? Declare only the ones we want to expose or a folder?

Reply:
I tried to do the opposite but I had a problem with the flyway. 
Sure there will be a way but we will need to invest more time to it.